### PR TITLE
Responsive footer styles and add new images and icons

### PR DESCRIPTION
**Pull request by hectoruch**
Updates:

- Responsive styles for Footer.
- New share icon add and LDF Logo white for header and footer.

Pivotal Task: #129129743

*Need crossbrowser test.

### CODE REVIEW

**Comment by tiagojsag** [comment]
- [ ] "follow us" not alligned to grid
- [ ] "Funding Partners" text missing above LDF logo (ignore text on zeplin)
- [ ] LDF logo needs to scale down in the same proportion as the other logos
- [ ] 742 px width => split links space evenly

**Comment by hectoruch** [comment]
WIP => because we are waiting dani designs for the email button.

File: app/src/components/Shared/Header.jsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import baseStyle from '../../../styles/_base.scss';
 import betaLogo from '../../../assets/logos/gfw_logo_beta.svg';
 import defaultLogo from '../../../assets/logos/gfw_logo_hor.svg';
 import menuicon from '../../../assets/icons/menu_icon.svg';
+import share from '../../../assets/icons/share_icon.svg';
```
Comment (andresgnlez) [inline app/src/components/Shared/Header.jsx]: Keep nomenclature, please: `shareIcon` . Just `share` can be confuse.

File: app/styles/components/c-footer.scss [hunk 2]
```diff
@@ -7,54 +7,121 @@
...
+
+        @media #{$mq-laptop} {
+          margin-right: 55px;
+        }
+
+        .logo-ldf {
+          width: calc(100%/3);
+
+          @media #{$mq-mobile-large} {
+            max-width: none;
```
Comment (andresgnlez) [inline app/styles/components/c-footer.scss]: Is this necessary? You are not using `max-width` previously, there's no need to set it.

**Comment by andresgnlez** [comment]
@hectoruch 

image
on Firefox 48.0.2

and check IE 11 & Edge, please.

**Comment by andresgnlez** [comment]
image

Are those literals OK? (`founding` and `funding` ones)

**Comment by tiagojsag** [comment]
@andresgnlez use partners page as reference

**Comment by andresgnlez** [comment]
image

There are like 10px less compared with design.

**Comment by andresgnlez** [comment]
@tiagojsag ah, my bad then! No problem then!

File: app/styles/components/c-footer.scss [hunk 3]
```diff
@@ -198,10 +198,10 @@
 
   .social-section {
     align-items: flex-start;
-    border-top: 2px solid rgba($color-4, .2);;
+    border-top: 1px solid rgba($color-4, .2);;
```
Comment (andresgnlez) [inline app/styles/components/c-footer.scss]: dual semicolon

File: app/styles/components/c-footer.scss [hunk 4]
```diff
@@ -198,10 +198,10 @@
...
     display: block;
-    padding-bottom: 45px;
-    padding-top: 30px;
+    padding-bottom: 35px;
+    padding-top: 35px;
```
Comment (andresgnlez) [inline app/styles/components/c-footer.scss]: `padding: 35px 0` ?

Comment (hectoruch) [inline app/styles/components/c-footer.scss]: Yes, sorry.

**Comment by andresgnlez** [comment]
@hectoruch check changes out and merge.

**Pull request merged**